tools/clients/smpl_verts_client.py

```diff
@@ -0,0 +1,76 @@
+# yapf: disable
+import argparse
+import logging
+import numpy as np
+import os
+import sys
+import time
+from tqdm import tqdm
+
+from xrmocap.client.smpl_verts_client import SMPLVertsClient
+
+# yapf: enable
+
+
+def main(args) -> int:
+    name = os.path.basename(__file__).split('.')[0]
+    logger = logging.getLogger(name)
+    if args.verbose:
+        logger.setLevel(logging.INFO)
+    else:
+        logger.setLevel(logging.WARNING)
+    if args.smpl_data_path is None:
+        logger.error('Please specify smpl_data_path.')
+        raise ValueError
+    client = SMPLVertsClient(
+        server_ip=args.server_ip, server_port=args.server_port, logger=logger)
+    n_frames = client.upload_smpl_data(args.smpl_data_path)
+    logger.info(f'Motion of {n_frames} frames uploaded.')
+    faces = client.get_faces()
+    faces_np = np.array(faces)
+    logger.info(f'Get faces: {faces_np.shape}')
+    start_time = time.time()
+    for frame_idx in tqdm(range(n_frames)):
+        verts = client.forward(frame_idx)
+        verts_np = np.array(verts)
```

Comment:
As it will only be used when `frame_idx == 0`, `verts_np` can be removed into `if`?

Reply:
Fixed in [fd72f5b](https://github.com/openxrlab/xrmocap/pull/125/commits/fd72f5b8b53ce9a9e02a76203691d6b4a874f02b).